metaflow/plugins/aip/aip.py

```diff
@@ -1407,6 +1410,8 @@ def _create_metaflow_step_op(
         preceding_component_inputs: List[str],
         preceding_component_outputs_dict: Dict[str, dsl.PipelineParam],
     ) -> ContainerOp:
+        card_decos = [deco for deco in node.decorators if deco.name == "card"]
```

Comment:
It seems you must ahve tested this with a `METAFLOW_AIP_ADD_DEFAULT_CARD=false` and a Flow with a card?

Reply:
Yes.